pkg/landscaper/installations/executions/template/testdata/spifftemplate/component-descriptor-12.yaml

```diff
@@ -0,0 +1,212 @@
+component:
```

Comment:
this file is duplicated.
Can we put that in a separate directory for shared resources.

Reply:
we removed the duplicate